nncf/experimental/openvino_native/quantization/algorithms/min_max/openvino_backend.py

```diff
@@ -172,32 +175,59 @@ def _get_num_samples(num_samples, target_point: OVTargetPoint):
         return num_samples
 
     @staticmethod
-    def minmax_statistic_collector(nncf_graph: NNCFGraph,
-                                   target_point: OVTargetPoint,
-                                   quantizer_config: QuantizerConfig,
-                                   inplace: bool,
-                                   num_samples: int = None,
-                                   ) -> TensorCollector:
+    def get_statistic_collector(range_estimator_params: RangeEstimatorParameters,
+                                nncf_graph: NNCFGraph,
+                                target_point: OVTargetPoint,
+                                quantizer_config: QuantizerConfig,
+                                inplace: bool,
+                                num_samples: int = None) -> TensorCollector:
         reduction_shape, use_abs_max =\
             OVMinMaxAlgoBackend._get_reduction_shape_and_use_abs_max(nncf_graph, target_point,
                                                                      quantizer_config)
         _num_samples = OVMinMaxAlgoBackend._get_num_samples(num_samples, target_point)
-        return get_min_max_stat_collector(_num_samples, reduction_shape, use_abs_max, inplace)
 
-    @staticmethod
-    def mean_minmax_statistic_collector(nncf_graph: NNCFGraph,
-                                        target_point: OVTargetPoint,
-                                        quantizer_config: QuantizerConfig,
-                                        use_per_sample_stats: bool,
-                                        inplace: bool,
-                                        num_samples: int = None,
-                                        ) -> TensorCollector:
-        reduction_shape, use_abs_max = \
-            OVMinMaxAlgoBackend._get_reduction_shape_and_use_abs_max(nncf_graph, target_point,
-                                                                     quantizer_config)
-        _num_samples = OVMinMaxAlgoBackend._get_num_samples(num_samples, target_point)
-        return get_mean_min_max_stat_collector(_num_samples, reduction_shape,
-                                               use_abs_max, use_per_sample_stats, inplace)
+        try:
```

Comment:
Do we really need a `try-except` blocks in such a sensitive shared method?

Reply:
My idea is that a user that received such error could understand that the problem is in `range_estimator_params` and will try another params as a workaround

Reply:
Please check new way of error handling